src/Diagnostics/FxCop/CSharp/Design/CodeFixes/CA1052CSharpCodeFixProvider.cs

```diff
@@ -39,16 +36,37 @@ public sealed override FixAllProvider GetFixAllProvider()
             if (classDeclaration != null)
```

Comment:
regarding cancellationToken.ThrowIfCancellationRequested();

Shouldn't we just return here instead of throwing an exception?